sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -0,0 +1,166 @@
+        self.arms = []
+        if rospy.has_param('ra_sr_ur_robot_hw'):
+            self.arms.append('ra')
+        if rospy.has_param('la_sr_ur_robot_hw'):
+            self.arms.append('la')
+        if len(self.arms) == 0:
+            rospy.logerr("No arms detected, shutting down %s", rospy.get_name())
+            rospy.signal_shutdown("No arms detected")
+        self.subscriber = rospy.Subscriber("/sr_arm/release", Bool, self.pedal_sub)
+        self.rate = rospy.Rate(5)
+        rospy.spin()
+
+    def pedal_sub(self, data):
+        if True == data.data:
+            self.release_arm()
+
+    def call_arm_service(self, side, service_name, service_type, dashboard=True):
+        if dashboard:
+            service_string = "/" + side + "_sr_ur_robot_hw/dashboard/" + service_name
+        else:
+            service_string = "/" + side + "_sr_ur_robot_hw/" + service_name
+        try:
+            service_call = rospy.ServiceProxy(service_string, service_type)
+            response = service_call()
+            return response
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to '%s' failed for arm %s. %s", service_name, arm, e)
+
+    def startup_arms(self, arms):
+        for arm in arms:
+            self.call_arm_service(arm, "power_on", Trigger)
+        rospy.sleep(5)
+        for arm in arms:
+            self.call_arm_service(arm, "brake_release", Trigger)
+        rospy.sleep(5)
+
+    def check_arms_e_stops(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                while safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                    rospy.logwarn("Emergency stop button is still pressed for arm %s, please release", arm)
+                    safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+                    rospy.sleep(2)
+
+    def check_fix_arms_protective_stop(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.PROTECTIVE_STOP:
+                rospy.loginfo("Protective stop detected on: %s", arm)
+                self.call_arm_service(arm, "unlock_protective_stop", Trigger)
+
+    def check_fix_arms_fault(self, arms):
+        fault = False
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.FAULT:
+                fault = True
+                rospy.loginfo("Fault detected on: %s", arm)
+                rospy.loginfo("This can be caused by using the external e-stop. Have you released it?")
+                self.call_arm_service(arm, "restart_safety", Trigger)
+        return fault
```

Comment:
you're not really using fault var apart from returning it. Get rid of did and just return true or false depending on safetymode

Reply:
The logic is `if (arm_A_in_fault || arm_B_in_fault) return true; else return false`, so that if either (or both) arms need unlocking then we can trigger just one wait timer from the `return true`

Reply:
same with the next two comments